modules/programs/zsh.nix

```diff
@@ -60,6 +60,16 @@ let
         description = "History file location";
       };
 
+      ignorePatterns = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        example = literalExample ''[ "rm *" "pkill *" ]'';
+        description = ''
+          Do not enter command lines into the history list
+          if they match any one of the given shell patterns.
+        '';
```

Comment:
Sorry for all the nitpicks, could you provide a link to the official documentation about these patterns in the description?

Reply:
Seems no other option did so.